video_store.py

```diff
@@ -0,0 +1,153 @@
+import requests
+import datetime
+from flask import json
+
+class Video_store:
+    def __init__(self, URL = "https://retro-video-store-api.herokuapp.com", selected_video=None, selected_customer=None):
+        self.url = URL
+        self.selected_video = selected_video
```

Comment:
Very cool that you used this attribute to track the selected video!
Very few submissions had this functionality